src/commands/studio/module.ts

```diff
@@ -19,6 +19,8 @@ class Module extends IModule {
             .requiredOption("--packageKeys <packageKeys...>", "Exports only given package keys")
             .option("--includeDependencies", "Include variables and dependencies", "")
             .option("--excludeActionFlows", "Don't export action flows")
+            .deprecationNotice("`content-cli export packages` is deprecated and is expected to be removed in subsequent updates around: 01-09-2025.\n" +
```

Comment:
If we're providing these new commands to the users, I think we should first create an official documentation for them in the repo, since right now these commands are "hidden" in that sense.

Reply:
Looked into it and considering that this command doesn't really have much usage, would propose to leave it like this until we have a clear plan on how will we proceed with the documentation of those commands. Will see if others have an opinion on it.

Reply:
Which command doesn't have much usage? The Studio export? If that's what you meant, how do we know if it doesn't?

Reply:
Thanks! Then I guess it's fine to deprecate them.

Reply:
Extra note: the `axios` user-agent calls seem to be coming from content-cli too, but from the old `http-client.service.ts`. The export/import commands of Studio use the newer `HttpClientServiceV2` which always appends the user-agent as content-cli. That's extra assurance on them not being used by these commands but only by the single export/import ones.

Reply:
That's great! Thanks for checking! 🎉